src/libraries/System.Diagnostics.Process/tests/Helpers.cs

```diff
@@ -47,5 +47,27 @@ public static void DumpAllProcesses()
                 p.Dispose();
             }
         }
+
+        public static string? GetProcessUserName(Process p)
+        {
+            try
+            {
+                if (Interop.OpenProcessToken(p.SafeHandle, 0x8u, out var handle))
```

Comment:
8?

Reply:
that's [`TOKEN_QUERY`](https://docs.microsoft.com/en-us/windows/win32/secauthz/access-rights-for-access-token-objects). I didn't check what it exactly means since it was existing code, I just moved it here as I needed to reuse it.